src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.product-card {
```

Comment:
<img width="504" alt="image" src="https://user-images.githubusercontent.com/102411130/226838250-d828319e-097b-4979-8c57-f34777a6a4b4.png">
create files for bem blocks